SConscript

```diff
@@ -102,7 +102,8 @@ if env['python']:
         pyenv['SHLIBSUFFIX'] = '.pyd'
 
     pyenv.Append(SWIGFLAGS = ['-python', '-Wextra', '-Werror', '-v', '-O'], SWIGPATH = '#/inc')
-    pyenv.Append(CPPFLAGS = ['-Wno-strict-aliasing'])
+    if platform == 'posix':
+        pyenv.Append(CPPFLAGS = ['-Wno-strict-aliasing'])
```

Comment:
I have never actually seen this error ever produced by code.  It typically comes as result of converting a memory block to a struct through pointer manipulation. Although it can be done it is undefined behavior.

About the only time I can see this error being justified is when getting bytes from a network and moving them into a struct. Is this what is going on in this code?

Reply:
I can't repro this error with either gcc or clang anymore.  I'll remove it.